Sming/Arch/Esp8266/build.mk

```diff
@@ -40,6 +40,8 @@ OBJCOPY			 	:= $(TOOLSPEC)objcopy
 OBJDUMP			 	:= $(TOOLSPEC)objdump
 GDB				:= $(TOOLSPEC)gdb
 
+GCC_UPGRADE_URL := https://sming.readthedocs.io/en/latest/arch/esp8266/getting-started/eqt.html
```

Comment:
Should we also update the 'pre-release' note on this page? I've yet to find any issues with it :-)